integration/integration.cabal

```diff
@@ -70,6 +70,7 @@ common common-all
     UndecidableInstances
     ViewPatterns
 
+-- (sic!)
```

Comment:
hm, ormolu seems to have moved this way out of reach...

maybe you can add a comment at the top of the block mentioning the extension?  not as helpful, but better than this.  :)